addons/frag/CfgAmmoFragParameters.hpp

```diff
@@ -0,0 +1,795 @@
+};
+class ammo_Missile_rim162: ammo_Missile_MediumRangeAABase {
+    GVAR(skip) = 0;
+    GVAR(charge) = 138; // dg
+    GVAR(metal) = 252; // dg
+    GVAR(gurney_c) = 2400;
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium), QGVAR(large)};
+};
+class ammo_Missile_AMRAAM_C: ammo_Missile_MediumRangeAABase {
+    GVAR(skip) = 0;
+    GVAR(charge) = 70; // dg
+    GVAR(metal) = 128; // dg
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium)};
+};
+class ammo_Missile_AMRAAM_D: ammo_Missile_MediumRangeAABase {
+    GVAR(skip) = 0;
+    GVAR(charge) = 70; // dg
+    GVAR(metal) = 128; // dg
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium)};
+};
+class ammo_Missile_AA_R77: ammo_Missile_MediumRangeAABase {
+    GVAR(skip) = 0;
+    GVAR(charge) = 80; // dg
+    GVAR(metal) = 145; // dg
+    GVAR(gurney_c) = 2700;
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(large), QGVAR(large)};
+};
+
+class M_Vorona_HEAT: MissileBase {
+    // tandem shaped charges
+    GVAR(skip) = 1;
+};
+class M_Vorona_HE: M_Vorona_HEAT {
+    // All signs point to this being a thermobaric round so low frag count
+    GVAR(skip) = 0;
+    GVAR(metal) = 13800;
+    GVAR(charge) = 4950;
+    GVAR(gurney_c) = 2800;
+    GVAR(gurney_k) = "1/2";
+    GVAR(classes)[] = {QGVAR(tiny)};
+};
+
+class M_127mm_Firefist_AT: RocketBase { // HOT missile
+    GVAR(skip) = 1;
+};
+
+class M_NLAW_AT_F: MissileBase {
+    GVAR(skip) = 1;
+};
+
+// ~~~~ Shell
+class Sh_75mm_Railgun_APFSDS: ShellBase {
+    GVAR(skip) = 1;
+};
+class Sh_120mm_APFSDS: ShellBase {
+    GVAR(skip) = 1;
+};
+class Sh_125mm_APFSDS: ShellBase {
+    GVAR(skip) = 1;
+};
+
+class Sh_155mm_AMOS: ShellBase {
+    // Source: http://www.globalsecurity.org/military/systems/munitions/m795.htm
+    GVAR(skip) = 0;
+    GVAR(charge) = 9979;
+    GVAR(metal) = 36000;
+    GVAR(gurney_c) = 2440;
+    GVAR(gurney_k) = "1/2";
+    GVAR(classes)[] = {QGVAR(large), QGVAR(large), QGVAR(large_HD), QGVAR(large), QGVAR(huge), QGVAR(huge_HD), QGVAR(huge)};
+};
+
+class Sh_82mm_AMOS: Sh_155mm_AMOS { // VO-832DU
+    GVAR(charge) = 420;
+    GVAR(metal) = 2680;
+    GVAR(classes)[] = {QGVAR(medium), QGVAR(medium_HD)};
+};
+
+class Sh_120mm_HE: ShellBase {
+    GVAR(skip) = 0;
+    GVAR(charge) = 3148;
+    GVAR(metal) = 23000;
+    GVAR(gurney_c) = 2830;
+    GVAR(gurney_k) = "1/2";
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium), QGVAR(medium_HD)};
+};
+class Sh_120mm_HEAT_MP: ShellBase {
+    GVAR(skip) = 0;
+    GVAR(charge) = 2500;
+    GVAR(metal) = 5000;
+    GVAR(gurney_c) = 2500;
+    GVAR(gurney_k) = "1/2";
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium), QGVAR(medium_HD)};
+};
+
+class ammo_ShipCannon_120mm_HE: Sh_155mm_AMOS {
+    GVAR(charge) = 3148;
+    GVAR(metal) = 23000;
+    GVAR(gurney_c) = 2830;
+    GVAR(classes)[] = {QGVAR(small), QGVAR(small), QGVAR(medium), QGVAR(large)};
+};
+
+class Sh_125mm_HEAT: Sh_120mm_HE {
```

Comment:
```suggestion
class Sh_125mm_HEAT: Sh_125mm_HE {
```

Reply:
need to shuffle below 744

Reply:
Completed at https://github.com/acemod/ACE3/pull/10157/commits/58e021bda4455b6e3ce59eacb7269764fc7ba8f4